### DISCUSSION TITLE
Meu site no Vercel esta com um pequeno problema / My website on Vercel has a small problem

### DISCUSSION THREAD

**Discussion by migueloliveira4002** [vercel/community#4755] [Help]
### Summary

Quando eu envio meu site para o GitHub e depois importo para o Vercel, ele da esse erro que esta no site, alguem poderia me ajudar? o index.html esta na pasta "pages"

https://proj-extensao-01-git-main-migueloliveira4002.vercel.app/

When I uploaded my website to GitHub and then imported it into Vercel, it gives this error that is on the website, could anyone help me? index.html is in the "pages" folder

https://proj-extensao-01-git-main-migueloliveira4002.vercel.app/

### Example

https://proj-extensao-01-git-main-migueloliveira4002.vercel.app/

### Steps to Reproduce

![Cap01](https://github.com/vercel/community/assets/17089605/b28fda36-839f-43f5-9029-9cec64757992)
![Cap02](https://github.com/vercel/community/assets/17089605/be1a44ba-38d3-400c-b48f-43ddb70024e5)

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to get a better view of what is going wrong?
